File: web/app/model/sm/TargetCost.js
Ext.define('iFlat.model.sm.TargetCost', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'targetCost.id', mapping: 'id', type: 'string'},
        {name: 'targetCost.type', mapping: 'type', type: 'string'},
        {name: 'targetCost.projNo', mapping: 'projNo', type: 'string'},
        {name: 'targetCost.projName', mapping: 'projName', type: 'string'},
        {name: 'targetCost.costAccount', mapping: 'costAccount', type: 'string'},
        {name: 'targetCost.costAccountName', mapping: 'costAccountName', type: 'string'},
        {name: 'targetCost.amount', mapping: 'amount', type: 'number'},
    ]
});

File: src/main/java/com/iflat/sm/bean/TargetCost.java
package com.iflat.sm.bean;

import java.util.Date;

/**
 * Created by tyriv on 2016/3/21.
 */
public class TargetCost {

    private String id;
    private String type;
    private String projNo;
    private String projName;
    private String costAccount;
    private String costAccountName;
    private double amount;
    private String creatorAcc;
    private String creatorName;
    private Date createTime;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getProjNo() {
        return projNo;
    }

    public void setProjNo(String projNo) {
        this.projNo = projNo;
    }

    public String getCostAccount() {
        return costAccount;
    }

    public void setCostAccount(String costAccount) {
        this.costAccount = costAccount;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public String getProjName() {
        return projName;
    }

    public void setProjName(String projName) {
        this.projName = projName;
    }

    public String getCostAccountName() {
        return costAccountName;
    }

    public void setCostAccountName(String costAccountName) {
        this.costAccountName = costAccountName;
    }
}
